test-one/src/index.js

```diff
@@ -2,13 +2,35 @@ import { log } from "./helpers/logger";
 
 export const go = async () => {
   // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
+  var message = "Result-Above";
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+  var test = "";
 
-  await log(welcomeMessage);
+  var i;
+  for(i = 1; i < 101; i++){
+
+    if (i % 3 == 0 && i % 5 == 0){
+      test += "BossHog" + "\n";
+    }
+
+    else if (i % 3 == 0) {
+      test += "Boss" + "\n";
+    }
+
+    else if (i % 5 == 0) {
+      test += "Hog" + "\n";
+    }
+
+    else{
+    test += String(i) + "\n";
```

Comment:
Are you aware of any plugin/library that would help with styling/whitespace inconsistency like we have here?